src/diffusers/pipelines/text_to_video_synthesis/pipeline_text_to_video_synth.py

```diff
@@ -634,6 +634,9 @@ def __call__(
         # 6. Prepare extra step kwargs. TODO: Logic should ideally just be moved out of the pipeline
         extra_step_kwargs = self.prepare_extra_step_kwargs(generator, eta)
 
+        # 6.1 Chunk feed-forward computation to save memory
+        self.unet.enable_forward_chunking(chunk_size=1, dim=1)
```

Comment:
Are these reasonable defaults or should we also the users to specify these too?

Reply:
Reasonable defaults, here this means that we run one forward pass per frame